### DISCUSSION TITLE
20. Mobile First

### DISCUSSION THREAD

**Discussion by dNicolle** [cnumr/best-practices-wordpress#33] [Bonnes pratiques] [answered]
https://github.com/cnumr/fiches-wordpress-greenit/blob/main/best-practices-WordPress/fiches/20.%20Mobile%20First.md

Il manque le principe de validation et le format ne me semble pas correspondre.

**Comment by dNicolle** [comment, marked as answer]
@hrenaud et @Yanngautreau  : Il faudrait également modifier le titre pour qu'il comporte un verbe

**Reply by hrenaud** [reply]
J'ai fait les changements
